app/src/main/java/com/example/solidarity/Event.java

```diff
@@ -0,0 +1,95 @@
+package com.example.solidarity;
+
+import android.text.format.DateUtils;
+
+import com.parse.ParseClassName;
+import com.parse.ParseFile;
+import com.parse.ParseObject;
+import com.parse.ParseUser;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+@ParseClassName("Event")
+public class Event extends ParseObject {
+
+    private static final String KEY_DESCRIPTION = "description";
+    private static final String KEY_IMAGE = "image";
+    private static final String KEY_AUTHOR = "author";
+    private static final String KEY_CREATED_KEY = "createdAt";
+    private static final String KEY_TITLE = "title";
+    private static final String KEY_LOCATION = "location";
+    private static final String KEY_DATE = "date";
+
+
+    public String getDescription() {
+        return getString(KEY_DESCRIPTION);
+    }
+
+    public void setDescription(String description) {
+        put(KEY_DESCRIPTION, description);
+
+    }
+
+    public ParseFile getImage() {
+        return getParseFile(KEY_IMAGE);
+    }
+
+    public void setImage(ParseFile parseFile) {
+        put(KEY_IMAGE, parseFile);
+    }
+
+    public ParseUser getAuthor() {
+        return getParseUser(KEY_AUTHOR);
+    }
+
+    public void setAuthor(ParseUser author) {
+        put(KEY_AUTHOR, author);
+    }
+
+    public String getTitle() {
+        return getString(KEY_TITLE);
+    }
+
+    public void setTitle(String title) {
+        put(KEY_TITLE, title);
+    }
+
+    public String getLocation() {
+        return getString(KEY_LOCATION);
+    }
+
+    public void setLocation(String location) {
+        put(KEY_LOCATION, location);
+    }
+
+    public Date gDate() {
+        return getDate(KEY_DATE);
+    }
+
+    public void setDate(Date date) {
+        put(KEY_DATE, date);
+    }
+
+
+    public static String getRelativeTimeAgo(String rawJsonDate) {
```

Comment:
This function gives a String which describes how long ago an event occurred based on any String input of a Date- I made it static since it doesn't use any instance variables. I will probably be using it in the event timeline page.